pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -79,7 +79,7 @@ type Manager interface {
 
 	// GetCPUs implements the podresources.CPUsProvider interface to provide allocated
 	// cpus for the container
```

Comment:
can we also add to the comment what the boolean is?

Reply:
Sure, will do!